x/authz/keeper/keeper.go

```diff
@@ -256,6 +256,31 @@ func (k Keeper) DeleteGrant(ctx context.Context, grantee, granter sdk.AccAddress
 	})
 }
 
+// DeleteAllGrants revokes all authorizations granted to the grantee by the granter.
+func (k Keeper) DeleteAllGrants(ctx context.Context, granter sdk.AccAddress) error {
+	var keysToDelete []string
```

Comment:
nit: no need to be a []string, make it a [][]byte please

Reply:
yeah. I've just updated